src/pages/TotalSchedule.tsx

```diff
@@ -41,41 +55,39 @@ export const TotalSchedule = () => {
               justifyContent="flex-start"
               alignItems="normal"
               gap={5}
-              css={css`
-                overflow-y: scroll;
-              `}
             >
               <FlexBox
                 alignItems="normal"
                 padding="0 0 10px 0"
                 gap={10}
                 css={css(`position: sticky; top: 0; z-index: 50; background-color: ${colors.WH};`)}
               >
-                {/* TODO: 완벽한 시간 이전 워딩 추가 필요 */}
-                <Body2 color="purple">째깍! 완벽한 시간이 도착했습니다!</Body2>
+                <Body2 color="purple">
+                  {isPastDate(meetingData.dueDateTime)
+                    ? '째깍! 완벽한 시간이 도착했습니다!'
+                    : '째깍! 딱 맞는 시간을 찾기 위해 일정을 수집하고 있어요.'}
+                </Body2>
                 <SegmentedControl
                   variant="contained"
-                  selectedValue={selectedValue}
-                  onChange={setSelectedValue}
+                  selectedValue={selectedValue === 'DEFAULT' ? '사람 많은 순' : '빠른 시간 순'}
+                  onChange={handleSegmentChange}
                 >
                   <SegmentedControl.Tab label="사람 많은 순" value="사람 많은 순" />
                   <SegmentedControl.Tab label="빠른 시간 순" value="빠른 시간 순" />
```

Comment:
요렇게 표현하면 됩니당. 갠적으로 `DEFAULT` 대신 조금 더 사람 많은 순이라는 의미를 담은 네이밍이면 좋을 것 같아요. 예를 들면 `POPULAR`나 api와 동일하게 `COUNT`로 가져가도 좋구용
```suggestion
                  selectedValue={selectedValue}
                  onChange={setSelectedValue} // handleSegmentChange 없어도 될거예요
                >
                  <SegmentedControl.Tab label="사람 많은 순" value="DEFAULT" />
                  <SegmentedControl.Tab label="빠른 시간 순" value="LATEST" />
```

Reply:
`COUNT` ,`LATEST` 로 변경했습니당